src/ZODB/tests/util.py

```diff
@@ -206,3 +219,117 @@ def clear_transaction_syncs():
     for this.
     """
     transaction.manager.clearSynchs()
+
+
+class _TimeWrapper(object):
+
+    def __init__(self, granularity=1.0):
+        self._granularity = granularity
+        self._lock = ZODB.utils.Lock()
+
+    def configure_fakes(self, fake_gmtime, fake_time):
+        assert isinstance(fake_time, mock.Mock)
+        assert isinstance(fake_gmtime, mock.Mock)
+        def incr():
+            global _current_time # pylint:disable=global-statement
+            with self._lock:
+                _current_time = max(_real_time(), _current_time + self._granularity)
+            return _current_time
+        fake_time.side_effect = incr
+
+        def incr_gmtime(*seconds):
+            if seconds:
+                assert len(seconds) == 1
+                now = seconds[0]
```

Comment:
What about replacing the above 2 lines with `now, = seconds`.

Reply:
I don't like relying on the side-effect of iterable unpacking that way.

Reply:
or `(now,) = seconds`

Reply:
How about a kwarg? `def incr_gmtime(seconds=None): if seconds:` It can still be passed positionally, and because the C version doesn't allow kwargs at all (it throws a TypeError; there's no way to spell optional-but-only-positional in pure Python AFAIK) it remains compatible.

Reply:
A default parm value is also fine for me (I dislike the `assert len(foo) == 1; bar = foo[0]` patten so much).